README.md

```diff
@@ -6,15 +6,15 @@
 - The approximate speed in Keboola Connection is `200 MiB / s` (if `gzip` disabled).
 - Manifest is created if needed. Original manifest's keys are preserved.
 - Header from CSV table is moved to manifest's `columns` key if input table is not headless.
-- CSV table must use [default format](https://help.keboola.com/storage/tables/csv-files/).
+- CSV delimiter and enclosure is loaded from manifest if set.
```

Comment:
Pridal som podporu pre nacitavanie `delimiter` a `enclosure` z manifestu. Tymto by to malo byt vsetko hotove.